src/maintainer/knowledge_base.rst

```diff
@@ -118,28 +118,55 @@ use a general web search to explore — while these topics are beyond the
 scope of this documentation, there are ample discussions on them on the broader
 Internet.
 
+To bootstrap a conda environment and install ``conda-build``, consider
+`miniforge<https://github.com/conda-forge/miniforge>`__.
+
+
+Executing a build
+^^^^^^^^^^^^^^^^^
+
+The ``build-locally.py`` script does not support Windows. Use ``conda build
+recipe/``.
+
+Using a local channel looks a bit different in Windows, too (note the
+triple-forward-slash in the channel name):
+
+.. code-block::
+
+    conda create -n my-new-env -c file:///C:\Users\User\miniforge3\conda-build my-package
+
+If you installed from a different distribution than ``miniforge`` or to a
+different path, you'll need to alter this path.
+
+
+Notes on native code
+--------------------
+
 In order to compile native code (C, C++, etc.) on Windows, you will need to
 install Microsoft’s Visual C++ build tools on your VM. You must install
 particular versions of these tools — this is to maintain compatibility between
 compiled libraries used in Python, `as described on this Python wiki page
 <https://wiki.python.org/moin/WindowsCompilers>`__. The current relevant
 versions are:
 
-* For Python 3.5–3.7: Visual C++ 14.0
+* For Python 3.5–3.10: Visual C++ 14.x
```

Comment:
Interesting how the Python wiki doesn't mention Python 3.11 yet...

Reply:
Indeed. I was treating that page as the source of truth; do you know if Visual C++ 14.x actually supports 3.11?  It's not an area I know much about, and my web search didn't turn up an answer.

Reply:
This has been updated now in the wiki and they do mention 3.12+, so:

```suggestion
* For Python 3.5–3.12+: Visual C++ 14.x
```